src/util/invariant.cpp

```diff
@@ -79,8 +79,6 @@ void print_backtrace(
   std::ostream &out)
 {
 #ifdef __GLIBC__
-    out << "Backtrace\n" << std::flush;
-
```

Comment:
Someone has finally noticed that it prints backtrace twice! :-)

Reply:
Or someone has finally bothered to fix it... Usually when you see this you have other worries than this ;)

Reply:
How very true...